system/group.py

```diff
@@ -112,11 +119,18 @@ def group_mod(self, **kwargs):
                 if kwargs[key] is not None and info[2] != int(kwargs[key]):
                     cmd.append('-g')
                     cmd.append(kwargs[key])
+            if key == 'force':
+                if kwargs[key] is True:
+                    curname = grp.getgrgid(kwargs[gid])[0]
+                    cmd.append('-n')
+                    cmd.append(self.name)
+                    cmd.append(curname)
         if len(cmd) == 1:
             return (None, '', '')
         if self.module.check_mode:
             return (0, '', '')
-        cmd.append(self.name)
+        if kwargs['force'] is False:
+            cmd.append(self.name)
```

Comment:
@b3rn, I think, it would be better to keep unconditional `cmd.append(self.name)` and remove it from *L127* since last parameter for `usermod` is always `GROUP`.